docs/vector/api_reference.rst

```diff
@@ -0,0 +1,140 @@
+.. automodule:: gym.vector
+    :noindex:
+
+API Reference
+=============
+
+.. autofunction:: make
+
+VectorEnv
+---------
+
+.. autoclass:: VectorEnv
+    
+    .. py:property:: action_space
+
+        :type: :class:`gym.spaces.Space`
+
+        The (batched) action space. The input actions of :meth:`step` must be valid elements of :obj:`action_space`.
+
+        .. code-block::
+
+            >>> envs = gym.vector.make("CartPole-v1", num_envs=3)
+            >>> envs.action_space
+            MultiDiscrete([2 2 2])
```

Comment:
Hmm. What happens when the origin env's action space is `MultiDiscrete`, do you return a tuple of `MultiDiscrete`?

Reply:
The rules for batching all the standard Gym spaces are available [here](https://github.com/openai/gym/blob/2853ce4797a4dadb1f8702c8e35143d9fbd43ea6/gym/vector/utils/spaces.py#L10). For example a `MultiDiscrete` space becomes a `Box` with an appropriate shape. Do you think it is worth mentioning these conversion rules in the documentation?